src/OrbitCaptureClient/CaptureClient.cpp

```diff
@@ -53,7 +53,7 @@ ErrorMessageOr<void> CaptureClient::StartCapture(
     ThreadPool* thread_pool, const ProcessData& process,
     const orbit_client_data::ModuleManager& module_manager,
     absl::flat_hash_map<uint64_t, FunctionInfo> selected_functions,
-    TracepointInfoSet selected_tracepoints, UserDefinedCaptureData user_defined_capture_data,
```

Comment:
we want to translate function set to a set of intrumented function ids at the time they become available. Note also that this provides consistency by design since it is clear that frame_track_ids should be in selected function map.